schema/Cargo.toml

```diff
@@ -8,6 +8,7 @@ publish = false
 
 [dependencies]
 diesel = { version = "1.4", optional = true, features = ["postgres", "serde_json"] }
+diesel_migrations = { version = "1.4.0", features = ["postgres", "sqlite"] }
```

Comment:
@tjsharp1 
- I'm assuming this is added `diesel_migrations` crate is responsible for the migration file changes [like this one here](https://github.com/FuelLabs/fuel-indexer/pull/94/files#diff-5ba27fcf941f34a3f45e0a5c999a2be4817bc9c1edd8593bd4b485be7dc693feR6) ?